game-app/game-core/src/main/java/games/strategy/triplea/attachments/AbstractRulesAttachment.java

```diff
@@ -179,10 +177,6 @@ private void resetGameProperty() {
   }
 
   private void setRounds(final String rounds) throws GameParseException {
```

Comment:
I assume rounds is not ever supposed to be null here, just like the other case where I asked?

Reply:
Correct, every setter is the same.